public/css/button15.css

```diff
@@ -1,6 +1,6 @@
 .button15 {
-    color: black;
-    background-color: white;
+    color: red;
+    background-color:gray;
```

Comment:
他とスタイルを合わせるためにコロンの後にはスペースがあると良いと思いました！

```suggestion
    background-color: gray;
```

Reply:
修正します！